docs/src/explanations/connectable.md

```diff
@@ -0,0 +1,444 @@
+
+First, we can use a similar example to `Parent` but use `Input/Output` instead of `Flipped`.
+Because `alignedChild` and `flippedChild` are non-aggregates, `Input` is basically just a `Flipped` and thus the alignments are unchanged compared to the previous `Parent` example.
+
+```scala mdoc:silent
+import chisel3._
+class ParentWithOutputInput extends Bundle {
+  val alignedCoerced = Output(UInt(32.W)) // Equivalent to just UInt(32.W)
+  val flippedCoerced = Input(UInt(32.W))  // Equivalent to Flipped(UInt(32.W))
+}
+class MyModule2 extends Module {
+  val p = Wire(new ParentWithOutputInput)
+}
+```
+
+The aligments are the same as the previous `Parent` example:
+ - `p` is aligned w.r.t `p`
+ - `p.alignedCoerced` is aligned w.r.t `p.alignedCoerced`
+ - `p.flippedCoerced` is aligned w.r.t `p.flippedCoerced`
+ - `p` is aligned w.r.t `p.alignedCoerced`
+ - `p` is flipped w.r.t `p.flippedCoerced`
+ - `p.alignedCoerced` is flipped w.r.t `p.flippedCoerced`
+
+The next example has a nested bundle `GrandParent` who instantiates an `Output` `ParentWithOutputInput` field and an `Input` `ParentWithOutputInput` field.
+
+```scala mdoc:silent
+import chisel3._
+class GrandParentWithOutputInput extends Bundle {
+  val alignedCoerced = Output(new ParentWithOutputInput())
+  val flippedCoerced = Input(new ParentWithOutputInput())
+}
+class MyModule3 extends Module {
+  val g = Wire(new GrandParentWithOutputInput)
+}
+```
+
+Remember that `Output(gen)/Input(gen)` recursively strips the `Flipped` of any recursive children.
+This makes every member of `gen` aligned with every other member of `gen`.
+
+Consider the following alignments between grandparent and grandchildren. Because `alignedCoerced` and `flippedCoerced` are aligned with all their recursive members, they are fully aligned. Thus, only their alignment to `g` influences grandchildren alignment:
+ - `g` is aligned w.r.t `g.alignedCoerced.alignedChild`
+ - `g` is aligned w.r.t `g.alignedCoerced.flippedChild`
+ - `g` is flipped w.r.t `g.flippedCoerced.alignedChild`
+ - `g` is flipped w.r.t `g.flippedCoerced.flippedChild`
+
+Consider the following alignment relationships starting from `g.alignedCoerced` and `g.flippedCoerced`. *Note that whether `g.alignedCoerced` is aligned/flipped relative to `g` has no effect on the aligned/flipped relationship between `g.alignedCoerced` and `g.alignedCoerced.alignedChild` or `g.alignedCoerced.flippedChild` because alignment is only relative to the two members in question! However, because alignment is coerced, everything is aligned between `g.alignedCoerced`/`g.flippedAligned` and their children*:
+ - `g.alignedCoerced` is aligned w.r.t. `g.alignedCoerced.alignedChild`
+ - `g.flippedCoerced` is aligned w.r.t. `g.alignedCoerced.flippedChild`
+ - `g.alignedCoerced` is aligned w.r.t. `g.flippedCoerced.alignedChild`
+ - `g.flippedCoerced` is aligned w.r.t. `g.flippedCoerced.flippedChild`
```

Comment:
```suggestion
- `g.alignedCoerced` is aligned w.r.t. `g.flippedCoerced`
 - `g.alignedCoerced` is aligned w.r.t. `g.alignedCoerced.alignedChild`
 - `g.flippedCoerced` is aligned w.r.t. `g.alignedCoerced.flippedChild`
 - `g.alignedCoerced` is aligned w.r.t. `g.flippedCoerced.alignedChild`
 - `g.flippedCoerced` is aligned w.r.t. `g.flippedCoerced.flippedChild`
```

This is *so* hard to wrap your head around, until you remember that "any root is aligned with itself, thus anyRoot is aligned w.r.t anyOtherRoot", which I think helps give better intuition to the lines below 🤯

Reply:
or no i dont think that is right, we would say that `g.alignedCoerced` is flipped w.r.t. `g.flippedCoerced` given the example above:


```
Finally, we can list all sibling relationships:
 - `p.alignedChild` is flipped w.r.t `p.flippedChild`
```

so there is something confusing going on here. I don't understand how both of these examples can be true. why do we even care in your set of examples the relationship between g.flippedCoerced and g.alignedCoered.flippedChild? I'm going to suggest a different change to this text